packages/mdc-list/foundation.js

```diff
@@ -301,7 +296,7 @@ class MDCListFoundation extends MDCFoundation {
         nextIndex = 0;
       } else {
         // Return early because last item is already focused.
-        return;
+        return nextIndex;
```

Comment:
This is incorrect and will return an out-of-bounds index. `index` is the old index, which is what should be returned, as I mentioned in my previous comment.

Same for `focusPrevElement` and `return prevIndex`.

This PR has 100% coverage but I have to assume we're not verifying these branches...